<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1368</width>
    <height>889</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="glwidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>931</width>
      <height>821</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_left">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Перемещение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_up">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_down">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_right">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_in">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>o</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_out">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Вращение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_turnPlusOX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+OX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_turnPlusOY">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+OY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_turnPlusOZ">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+OZ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Масштаб</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_zoomIn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_zoomOut">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_filename">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_turnMinusOZ">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-OZ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_turnMinusOY">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-OY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_turnMinusOX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-OX</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_settings_Flustrum">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Flustrum</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_settings_Normal">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Facets</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_settings_StippleLines">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stipple</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_settings_SolidLines">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Solid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_settings_FacetsColor">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>190</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Facets Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_settings_FacetsThickness">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>230</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Facets Thickness</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dots</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_settings_Dots_SquareMode">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>300</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Square</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_settings_Dots_Circlemode">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>300</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_settings_Dots_VoidMode">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>300</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Void</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_settings_DotsSize">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>340</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Dots Size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_settings_DotsColor">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>380</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Dots Color</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>410</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>BackGround</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_settings_BGColor">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>440</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Change BG Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_MakeScreenshot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Make screenshot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_MakeGif">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Make Gif</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_settings_Reset">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>730</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1368</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glwidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">glwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
